### DISCUSSION TITLE
Account instantly flagged upon creation

### DISCUSSION THREAD

**Discussion by Spazmatism-Eye** [community/community#60167] [New to GitHub] [answered]
My friend made a GitHub account for the first time so we could work on something together. However, his account was instantly flagged as soon as he created it, and before he could do anything or I could add him to the repo.

**Comment by GhostOf0days** [comment, marked as answer]
Hi. Please ask him to reach out to [Github Support](https://support.github.com/) if possible on that account. He can create a ticket by clicking 'My Tickets' in the top right.

**Reply by Spazmatism-Eye** [reply]
It only lets him create a ticket under the "Billing" category.

**Reply by GhostOf0days** [reply]
Just do that then. They should still be able to answer it.

**Reply by runi293** [reply]
yes my acuant
